exercises/concept/card-games/.docs/instructions.md

```diff
@@ -0,0 +1,101 @@
+# Instructions
+
+## Instructions
+
+Elyse is really looking forward to playing some poker (and other card games) during her upcoming trip to Vegas. Being a big fan of "self-tracking" she wants to put together some small functions that will help her with tracking tasks and has asked for your help thinking them through.
+
+## 1. Tracking Poker Rounds
+
+Elyse is especially fond of poker, and wants to track how many rounds she plays - and _which rounds_ those are. Every round has its own number, and every table shows the round number currently being played. Elyse chooses a table and sits down to play her first round. She plans on playing three rounds.
+
+Implement a function `to_rounds` that takes the current round number and returns a single list with that round and the _next two_ that are coming up:
```

Comment:
I had to read this a couple times to understand what `to_rounds` was doing. What about `get_rounds`?

Reply:
Agree.  `get_rounds` reads better.  @IsaacG ?

Reply:
No preference. Updating...